scripting/get5.sp

```diff
@@ -686,35 +686,49 @@ public Action Timer_CheckReady(Handle timer) {
 
 static void CheckReadyWaitingTimes() {
   if (g_TeamTimeToStartCvar.IntValue > 0) {
-    CheckReadyWaitingTime(MatchTeam_Team1);
-    CheckReadyWaitingTime(MatchTeam_Team2);
+    g_ReadyTimeWaitingUsed++;
+
+    bool team1Forfeited = CheckReadyWaitingTime(MatchTeam_Team1);
+    bool team2Forfeited = CheckReadyWaitingTime(MatchTeam_Team2);
+
+    if (team1Forfeited && team2Forfeited) {
+      g_ForcedWinner = MatchTeam_TeamNone;
+      Stats_Forfeit(MatchTeam_Team1);
```

Comment:
I think multiple calls to Stats_Forfeit yields misleading resulting.

As written, this will end up calling `Stats_SeriesEnd` for both teams and `STAT_SERIESWINNER` will be one of the teams. It should probably be mapping to the none team in that case.